src/main/webapp/script.js

```diff
@@ -0,0 +1,22 @@
+/** Used for accessing form input values. */
+const LECTURE_FORM = 'lectureForm';
+const NAME_INPUT = 'name-input';
+const LINK_INPUT = 'link-input';
+
+/** Used to send an error message to client. */
+const LECTURE_NAME_ERROR = 'Lecture name must be filled out.';
+const VIDEO_URL_ERROR = 'Video url must be filled out.';
+
+/** Checks if lecture form is filled entirely, sends an alert if not. */
+function validateForm() {
```

Comment:
Try marking the inputs as required, I might be wrong, but it should work as you'd expect and this function can be removed

Reply:
removed